testdata/jennies/builders/constraints/JavaBuilders/constraints/SomeStruct.java

```diff
@@ -12,19 +12,19 @@ public Builder() {
             this.internal = new SomeStruct();
         }
     public Builder setId(Long id) {
-        if (id >= 5) {
-            return this;
+        if (!(id >= 5)) {
+            throw new RuntimeException("id must be >= 5");
```

Comment:
nit: These feel more like `IllegalArgumentException`: https://docs.oracle.com/javase/8/docs/api/java/lang/IllegalArgumentException.html which is a subclass of RuntimeException